pkg/pcrbruteforcer/reproduce_expected_pcr0_test.go

```diff
@@ -0,0 +1,181 @@
+package pcrbruteforcer
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"testing"
+
+	"github.com/9elements/converged-security-suite/v2/pkg/pcr"
+	"github.com/9elements/converged-security-suite/v2/pkg/registers"
+	"github.com/9elements/converged-security-suite/v2/pkg/uefi"
+	"github.com/linuxboot/contest/pkg/xcontext/bundles/logrusctx"
+	"github.com/linuxboot/contest/pkg/xcontext/logger"
+	"github.com/linuxboot/fiano/pkg/intel/metadata/manifest"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/linuxboot/contest/pkg/xcontext"
+	"github.com/stretchr/testify/require"
+)
+
+func getFirmware() *uefi.UEFI {
+	// It appears Y3DLD11 has wrong order of elements inside BPM:
+	// PFRE goes PCDE, thus disabling the host strict order check.
+	//
+	// TODO: make offsets stable to elements reordering, currently
+	// if position of PMSE will be wrong it will show a wrong area
+	// in the report.
+	manifest.StrictOrderCheck = false
+
+	firmwareImage, err := decompressImage("firmwares", "F0E_3A10.bin.xz")
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Is this worth a panic?

Reply:
I would'be pass `*testing.T`, but this also used for benchmarks, so what way do we have to terminate an error?